javasphinx/compiler.py

```diff
@@ -116,6 +116,14 @@ def compile_type(self, declaration):
 
         return directive
 
+    def compile_package_documentation(self, declaration):
```

Comment:
Since this method is really just compiling the docblock (the signature bits are going unused here), lets simplify this method and give it a new name,

```python
def compile_docblock(self, documented):
    ''' Compiles a single, standalone docblock. '''
    return self.__output_doc(documented).build()
```

Could you also move it just before or after the `compile` method to keep the public methods together? These other `compile_*` methods are internal.